mitmachen.html

```diff
@@ -102,9 +102,9 @@ <h3>Soziale Netze</h3>
         <h3>Noch Fragen?</h3>
         <p>
           Schreib' uns eine <a href="mailto:kontakt@pep-dortmund.org">Email</a> oder ruf' uns an <br>
-          Kevin Schmitz <strong>0231/755-2988</strong>,<br>
-          Karl Schiller <strong>0231/755-2151</strong>,<br>
-          Lars Bollmann <strong>0231/755-8503</strong>.
+          Christian Arauner <strong>0231/755-4534</strong>,<br>
+          Rene-Marcel Lehner <strong>0231/755-4534</strong>,<br>
```

Comment:
is this right, Rene is in the same office as Christian and Anno?

Reply:
Rene has no office, I just but the same as Anno's and CHristian's to avoid the blank space

Reply:
let me know if you prefer the blank space over this